pkg/ebpftracer/c/tracee.bpf.c

```diff
@@ -2103,27 +1948,18 @@ CGROUP_SKB_HANDLE_FUNCTION(proto)
                     dest = &nethdrs->protohdrs.udphdr;
                     size = get_type_size(struct udphdr);
                     break;
-                case IPPROTO_ICMP:
-                    dest = &nethdrs->protohdrs.icmphdr;
-                    size = 0; // will be added later, last function
-                    break;
                 default:
-                    return 1; // other protocols are not an error
+                    size = 0;
+                    break;
             }
 
-            neteventctx->md.flow.tuple.saddr.v4addr = nethdrs->iphdrs.iphdr.saddr;
```

Comment:
Looks like this whole `md.flow` flow struct and field can be also removed now

Reply:
The flow has been dropped 👌